rules-reviewed/technology-usage/tests/observability.windup.test.xml

```diff
@@ -20,6 +20,17 @@
                <fail message="Expected data not found for rule observability-0100"/>
             </perform>
          </rule>
+         <rule id="observability-0200-test">
+            <when>
+               <not>
```

Comment:
can we specify the number of iterations we expect here please? Just to ensure that both conditions tested in the 'OR' clause are actually firing.

Reply:
Thanks @m-brophy I have made the changes requested